### DISCUSSION TITLE
"My rules" (dynamic filtering rules)

### DISCUSSION THREAD

**Discussion by Obegg** [uBlockOrigin/uBlock-issues#3226] [General] [answered]
I'm sorry, but I don't understand the dynamic filtering rules, and yes I did read the wiki - https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-quick-guide
My question is what difference does it make if I delete everything (the default values) in dynamic filtering rules? If I delete everything - does it hurt the blocking capabilities? I'm trying to make sense of this list like why would I need it at all?
Specifically speaking, I'm talking about this:
```
no-csp-reports: * true
no-large-media: behind-the-scene false
behind-the-scene * * noop
behind-the-scene * 1p-script noop
behind-the-scene * 3p noop
behind-the-scene * 3p-frame noop
behind-the-scene * 3p-script noop
behind-the-scene * image noop
behind-the-scene * inline-script noop
```

**Comment by Yuki2718** [comment]
Doesn't hurt blocking capability, but you may encounter problems once you added any global blocking rule (more so if you have other extensions). Paranoid user may want to micro-manage `behind-the-scene` rules though.

**Reply by Obegg** [reply]
Thank you for the fast reply!
I have no idea what those `behind-the-scene` rules, so basically it's better to keep those and not delete anything?

**Reply by Yuki2718** [reply, marked as answer]
> so basically it's better to keep those and not delete anything?

Sure.

**Reply by Obegg** [reply]
Thank you very much for your time and explanation!

**Reply by uBlock-user** [reply]
>  I have no idea what those behind-the-scene rules, so basically it's better to keep those and not delete anything?

If you remove these rules, then connections to filter lists servers will be blocked, this is one example.

**Reply by Obegg** [reply]
Thank you for further explanation! I won't be removing those.

**Reply by garry-ut99** [reply]
https://github.com/gorhill/uBlock/wiki/Behind-the-scene-network-requests
